runbotics/runbotics-scheduler/src/utils/pipes/schemas/start-process.schema.ts

```diff
@@ -1,5 +1,6 @@
-import { object } from 'yup';
+import { object, string } from 'yup';
 
 export const startProcessSchema = object({
     variables: object().optional(),
+    notificationUrl: string().optional(),
```

Comment:
I would name it more like `callbackUrl` its common in API convention. Its one time call per one process execution.